src/Frontend.hs

```diff
@@ -73,6 +73,15 @@ runFrontend cfg = do
     let logMsg = log ^. msgDaemonSend
     withPersist logMsg cfg (runFrontend' cfg logMsg)
 
+runBoostrap :: Config -> Action () -> IO ()
+runBoostrap cfg action = do
```

Comment:
Why is this function in `src/Frontend.hs`?  Shouldn't it be in the executable module?